dbclient/SecretsClient.py

```diff
@@ -132,37 +139,28 @@ def import_all_secrets(self, log_dir='secret_scopes/'):
                     create_scope_args['initial_manage_principal'] = 'users'
                 other_permissions = self.get_all_other_permissions(scope_name, scopes_acl_dict)
                 create_resp = self.post('/secrets/scopes/create', create_scope_args)
-                print(create_resp)
+                logging_utils.log_reponse_error(error_logger, create_resp, None, ['RESOURCE_ALREADY_EXISTS'])
```

Comment:
can we put `error_ignore_list=['RESOURCE...']`?

Reply:
✔️